src/extensions/askUserForConsent.ts

```diff
@@ -64,6 +65,33 @@ export async function displayRoles(
   utils.logLabeledBullet(logPrefix, message);
 }
 
+/**
+ * Displays APIs that will be enabled for the project and corresponding descriptions.
+ * @param extensionName name of extension to install/update
+ * @param projectId ID of user's project
+ * @param apis APIs that require user approval
+ */
+export async function displayApis(
+  extensionName: string,
+  projectId: string,
+  apis: extensionsApi.Api[]
+): Promise<void> {
+  if (!apis.length) {
+    return;
+  }
+  const question = `${clc.bold(
+    extensionName
+  )} will enable the following APIs for project ${clc.bold(projectId)}`;
+  const results: string[] = await Promise.all(
+    apis.map((api: extensionsApi.Api) => {
+      return `- ${api.apiName}: ${api.reason}`;
+    })
+  );
+  results.unshift(question);
+  const message = _.join(results, "\n");
```

Comment:
We've been trying to move away from using lodash (_), because its bloats our dependency size, and a lot of the functionality it offers has been added to Typescript. 

Prefer results.join("\n")

Reply:
Done.